### PR TITLE
Many enhancements + support chrome 99

### PR DESCRIPTION
**Pull request by Blondy314**

### CODE REVIEW

File: src/DownloadForm.Designer.cs [hunk 1]
```diff
@@ -43,6 +43,7 @@ private void InitializeComponent()
             this.OverallProgress_Label = new Guna.UI2.WinForms.Guna2HtmlLabel();
             this.EpisodeLoad_Label = new Guna.UI2.WinForms.Guna2HtmlLabel();
             this.EpisodeDonwload_Label = new Guna.UI2.WinForms.Guna2HtmlLabel();
+            this.txtUrl = new System.Windows.Forms.TextBox();
```
Comment (yairp03) [inline src/DownloadForm.Designer.cs:46]: The URL alone without the cookie is not worth much, I don't know if there's a point in adding it.

**Review by yairp03 — CHANGES_REQUESTED** [review]
The URL on the download form is a bit useless on its own, I think it should be removed or you can find a way to make it useful.
Besides that, looks great! Can't wait to merge this PR.
Glad you helping me revive this old project ;)

**Comment by Blondy314** [comment]
Hi,

Removed Download URL and added a few more improvements, hope u like it and
merge :)

Inbar.

File: src/DownloadForm.Designer.cs (hunk 1 above)
Comment (Blondy314) [reply src/DownloadForm.Designer.cs]: Removed

**Comment by Blondy314** [comment]
Any update on merging this?

**Comment by Blondy314** [comment]
Hi, pinging on this :)

**Comment by yairp03** [comment]
Hi, sorry for the late reply.
I've been busy lately, so I haven't had the chance to review the changes,
hopefully I can do so this week and approve the PR.

**Comment by Blondy314** [comment]
Pinging again, before they close the site ;)

**Pull request merged**